### DISCUSSION TITLE
尊敬的编写者们，深深的感谢你们编写的教程，顺便求一下你们的主题设置

### DISCUSSION THREAD

**Discussion by pandaxgc** [NakanoMikuOrg/arch-guide#197] [Q&A] [answered]
感谢你们制作的精美教程，我几乎一次就安装成功了，使用上了archlinux，同时着迷于你们教程的透明模糊的窗口装饰元素，尤其是zsh教程那篇的终端，非常好看，整体感很强，如果能分享一下主题的话感激不尽！🙏🏻🙏🏻😍😍
![lolcat 4be8f9f1](https://github.com/nakanomikuorg/arch-guide/assets/40298249/9227e02d-ce63-4652-aec7-1b16fa298bcd)
![chrome 7d1691f5](https://github.com/nakanomikuorg/arch-guide/assets/40298249/d423ab4d-167c-485f-ab57-f0332f867697)

**Comment by wuhang2003** [comment, marked as answer]
窗口主题参考 #147

**Comment by imroxe** [comment]
很遗憾，我没旱点发现这个项目，许多东西都是自己到处找资料，在找资料这个过程中，wiki不得不说很强大，基本上能解决大部分问题，即便有小部分问题，按照wiki的方向去查找解决方法，依然很不错。只是没这个项目那么全面，比如有了archinstall，再也不用自己去手动分区了，不过，手动有手动的好处。

在这，给大家推荐一个桌面环境，[HyDE](https://hydeproject.pages.dev/zh/)，hyprland，应该听过吧，我喜欢这种平铺式的桌面，这样会让自己更专注于某件事，不会因为窗口过多而分心，感兴趣的可以尝试

<img width="1920" height="1080" alt="图片" src="https://github.com/user-attachments/assets/deaaaa1f-987c-4ee4-abf5-9647d76a8dad" />
